src/cli/sdk.toit

```diff
@@ -405,26 +405,47 @@ Chooses the download URL based on the current platform.
 sdk-url version/string -> string:
   arch := system.architecture
   platform := system.platform
-  platform-str/string := ?
-  if platform == system.PLATFORM-LINUX:
+  selector/string := ?
+  if (semver.compare version "v2.0.0-alpha.163") < 0:
+    if platform == system.PLATFORM-LINUX:
+      if arch == system.ARCHITECTURE-X86-64:
+        selector = "linux"
+      else if arch == system.ARCHITECTURE-ARM64:
+        selector = "aarch64"
+      else if arch == system.ARCHITECTURE-ARM:
+        selector = "rpi"
+      else:
+        throw "Unsupported architecture: $platform-$arch"
+    else if system.platform == system.PLATFORM-MACOS:
+      selector = "macos"
+    else if system.platform == system.PLATFORM-WINDOWS:
+      selector = "windows"
+    else:
+      throw "Unsupported platform: $platform"
+  else:
+    if platform == system.PLATFORM-LINUX:
```

Comment:
I think I'd compute the platform and architecture strings separately (maybe by looking them up in two maps) and then I'd put together the selector after that (instead of the +=).

Reply:
done, but I left the 'if'.  There are some checks whether the combination is supported and that wouldn't work nicely with maps.